packages/bpk-component-text/src/BpkText.js

```diff
@@ -35,7 +45,17 @@ TEXT_STYLES.forEach(textStyle => {
 
 type Props = {
   children: Node,
-  textStyle: 'xs' | 'sm' | 'base' | 'lg' | 'xl' | 'xxl',
+  // eslint-disable-next-line flowtype/space-after-type-colon
```

Comment:
Had to add this lint rule here as the list is too long, which causes `prettier` to format it as below but then causes a `lint` error which also tries to format the code which then violates a `prettier` rule - meaning there is a circular error loop of rules conflicting